src/models/WidgetEventCapability.ts

```diff
@@ -25,14 +31,15 @@ export class WidgetEventCapability {
     private constructor(
         public readonly direction: EventDirection,
         public readonly eventType: string,
-        public readonly isState: boolean,
+        public readonly kind: EventKind,
         public readonly keyStr: string | null,
         public readonly raw: string,
     ) {
     }
 
-    public matchesAsStateEvent(eventType: string, stateKey: string): boolean {
-        if (!this.isState) return false; // looking for state, not state
+    public matchesAsStateEvent(direction: EventDirection, eventType: string, stateKey: string): boolean {
```

Comment:
I am assuming this is a breaking change?

Reply:
Yes, it is. I've asked Travis, and it seems some light breakages like this are okay given the current maturity of the package